resources/kcp/charts/component-reconcilers/templates/deployment.yaml

```diff
@@ -50,6 +50,13 @@ spec:
         {{- ($.Values.global.mothership_reconciler.features) | nindent 8 }}
         {{- end }}
         {{- end }}
+        {{- if $.Values.component_reconcilers }}
```

Comment:
We are going to introduce per-component configuration in another [PR](https://github.com/kyma-project/control-plane/pull/1601).
Nevertheless I will aprove that to bring that into the code, so that we can unify our approaches later - it would be good if all the configuration for given component is located in a single place in the .Values/overrides structure.